tap_frontapp/schemas/conversations.json

```diff
@@ -0,0 +1,54 @@
+{
+    "type": ["null", "object"],
+    "selected": true,
```

Comment:
All instances of `selected` should be removed from schemas, it's not a valid JSON Schema key. Stream and field selection is now performed via metadata in the catalog.